test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -90,6 +92,20 @@ def test_successful_purchase_with_american_express
     assert_equal 'accredited', response.message
   end
 
+  def test_successful_purchase_with_taxes_and_net_amount
+    # Minimum transaction amount is 0.30 EUR or ~1112 $COL on 1/27/20.
+    # This value must exceed that
+    @amount = 10000_00
```

Comment:
Should this be a normal variable and not an instance variable?

What does the underscore in this value indicate? Is it different than a decimal?

Reply:
You are right, this could be an instance variable since no other places are using it. The underscore is a visual helper only where `10000_00 == 1000000`. Normally it is used to break up large numbers where commas would go, i.e. `10_000_000` makes easily see that it is 10 million. Similarly since this is on cents, `10000_00` makes it easy to quickly see that this would be 10 thousand Colombian pesos.